changelog/645.doc.rst

```diff
@@ -0,0 +1 @@
+Fix the grammatical error in :class:``Guild`` channel properties.
```

Comment:
```suggestion
Fix grammatical errors in :class:`Guild` channel properties.
```
missed this earlier, my bad

Reply:
Fixed in https://github.com/DisnakeDev/disnake/pull/645/commits/74577147f63c5d20cfd6fd8f7c5944727b0f039f.